tools/minikube/scripts/start_pods.sh

```diff
@@ -27,6 +27,21 @@ if [[ -z "${PINAKES_CONTROLLER_VERIFY_SSL}" ]]; then
   exit 1
 fi
 
+# Set the environment variable for accessing insights service
+# export PINAKES_INSIGHTS_TRACKING_STATE=True|False
+# export PINAKES_INSIGHTS_URL=<<your insights url>>
+
+# Check if metrics collection is turned on
+PINAKES_INSIGHTS_TRACKING_STATE=${PINAKES_INSIGHTS_TRACKING_STATE:-False}
+PINAKES_INSIGHTS_URL=${PINAKES_INSIGHTS_URL:-}
+
+if [[ "${PINAKES_INSIGHTS_TRACKING_STATE}" = "True" ]]; then
```

Comment:
@hsong-rh I think this whole check can be removed.
The default values for
* PINAKES_INSIGHT_TRACKING_STATE = False
* PINAKES_INSIGHTS_URL = https://cert.cloud.redhat.com/api/ingress/v1/upload

Reply:
For dev/qe testing they would set
* export PINAKES_INSIGHT_TRACKING_STATE = True
* export PINAKES_INSIGHTS_URL  = https://stage_uploader_url

Reply:
In Production they only need to set
* export PINAKES_INSIGHT_TRACKING_STATE = True